tools/internal_ci/windows/grpc_run_tests_matrix.bat

```diff
@@ -20,6 +20,8 @@ If Not "%RUN_TESTS_FLAGS%"=="%RUN_TESTS_FLAGS:python=%" (
 )
 call tools/internal_ci/helper_scripts/prepare_build_windows.bat || exit /b 1
 
+python3 tools/run_tests/start_port_server.py
```

Comment:
This is fine for now as a hotfix, but:
- we should add a comment that starting port server in advance is necessary on windows and link to a bug. Like this everyone will forget about the problem in no time and we'll risk reintroducing the same issue later.
- we need to check that this is really the only entry point for run_tests.py tests.

We should address those as a followup.

Reply:
1. Addressed in https://github.com/grpc/grpc/pull/28012
2. If there were other entrypoints, I would expect to see deterministic failures in those jobs, but I don't see any such failures on `go/grpc-fusion-master`.